src/DynamoCore/Nodes/Attributes.cs

```diff
@@ -208,4 +208,26 @@ public NodeObsoleteAttribute(string descriptionResourceID, Type resourceType)
             }
         }
     }
+
+    [AttributeUsage(AttributeTargets.Class)]
+    public class InputParametersAttribute : Attribute
```

Comment:
what do you think about making these attributes tuple<string,type> that way we can display the type... just displaying the name is not that useful as the names are already present on the node, correct?

I'm also worried about the input parameters and input ports getting out of sync, potentially you could use get the name from the ports?